app/views/comments/_new.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_for :comment, url: comments_path do |f| %>
+  <%= f.text_area :text %><br />
+  <%= f.hidden_field :commentable_id, :value => idea_id %>
+  <%= f.hidden_field :commentable_type, value: @commentable_type %>
```

Comment:
The new hash syntax (line 4) is generally preferred. Normally I wouldn't comment to say this, but it's jarring to see inconsistent use on two neighboring lines.